torch/onnx/_internal/registration.py

```diff
@@ -33,26 +33,27 @@ def _dispatch_opset_version(
     """
     if not registered_opsets:
         return None
-    registered_versions = sorted(registered_opsets)
+
+    reversed_registered_versions = sorted(registered_opsets, reverse=True)
```

Comment:
nit: ascending or descending for clarity wdyt?

Reply:
Sure!

Reply:
Done